web/ui/react-app/src/pages/targets/target.ts

```diff
@@ -2,7 +2,7 @@ export interface Labels {
   [key: string]: string;
 }
 
-export interface Target {
+export type Target = {
```

Comment:
Is there a specific reason for changing from `type` to `interface` here?

Reply:
Ok I read some more articles on `type` vs. `interface` and there's indeed very few differences nowadays. I'm only mildly concerned about the inconsistency of having `type Target`, but `interface DroppedTarget`, etc. Now `Target` is the odd one out. Not sure if it's worth keeping it an interface though if it introduces more hassle somewhere else.

Reply:
I don't really have an opinion on this topic. It's quite odd for me you have two differents ways to define an object.
I basically changed it because it was conveniant, but is it a really good idea, that's good question :')